db/schema.rb

```diff
@@ -1013,17 +1014,13 @@
     t.datetime "finished_at", precision: nil
     t.text "error"
     t.integer "error_event", limit: 2
```

Comment:
not sure about these?

Reply:
I don't actually know what these changes are, all I've done is migrated. I've migrated and rolled back a few times, might try rebuilding my database.